.github/workflows/CI.yml

```diff
@@ -58,6 +71,7 @@ jobs:
           git --version
           node --version
           npm --version
+          npx commitlint --version
```

Comment:
@realmarv isn't this already done by commitlint.sh?

Reply:
Done